src/jobs/deploy_env.yml

```diff
@@ -34,7 +37,7 @@ steps:
       working_directory: << parameters.helm_dir >>
       command: |
         sed -i "s/appVersion: \".*\"/appVersion: \"${APP_VERSION}\"/g" << parameters.chart_name >>/Chart.yaml
-        aws secretsmanager get-secret-value --secret-id ${AWS_SECRET_NAME} | jq -r .SecretString | \
+        << parameters.retrieve_secrets >> \
```

Comment:
This also needs a change to line 44 `--values - \` - which instructs helm to read values from stdin.

Reply:
Also, we need to be careful with these sort lines - to ensure nobody accidentally prints them to stdout.

Reply:
good spot